### DISCUSSION TITLE
Twilio Alarms

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1393] [Office hours] [answered]
The specific questions I have are...

- Locations of the Create new alarm button
- Looking for a suggestion on the interaction or IA pattern for the toast component after an user creates an alarm
- Feedback on Paste components usage(Menu, data table, radio button and more icon) on the Manage alarms and Create new alarm pages

**Background**: The Alarm setup page will help customers to create an alarm. Not only Error logs but also other  products such as Messaging Insights. Currently we are testing the prototype with customers and gather their feedback
Research: [Research Report](https://docs.google.com/document/d/13EolwAo_k8SBtXggCu-6IApxbjkhGlpuSxIlYJ3TVD8/edit?usp=sharing)
Links to mockups or prototype: [Twilio Alarms](https://www.figma.com/file/3GaINOa9Epx7ENTZslfaWa/Alerts-Q1-2021?node-id=1607%3A0)

**Comment by zahnster** [comment]
@sang-yeo thank you for your post! Looking forward to having the team discuss this with you at Office Hours tomorrow.

**Comment by serifluous** [comment]
Hey @sang-yeo !

> Locations of the Create new alarm button

From the research, it seems research participants were able to succeed in the task to create a new alarm. Was there a specific concern or question you had about the placements?

You may want to consider adding a max width on the entire content area, so that the link/button doesn't go too far to the right on a very wide screen.

> When a user does same action from two different places where the user lands after finished the action? In this case, create an alarm. 

I'd recommend landing the user on the same page where they clicked the "Create new alarm" link, unless you had some indication or insight into whether customers would want to land somewhere else or didn't need to do anything else on the error log properties page, for example.

> Feedback on Paste components usage (Menu, data table, radio button and more icon) on the Manage alarms and Create new alarm pages

Added comments in Figma!

**Comment by nkrantz** [comment, marked as answer]
Hey @sang-yeo, just following up on today's Office Hours. [Here's the link to the annotated recording](https://www.loom.com/share/ef27bf910d2548aba497a5e85e45b992) (password is in slack). Thanks for coming!

**Reply by sang-yeo** [reply]
Thank you. I have one question regarding the 'Create new alarm' button. I use an anchor to take the user to the setup page. I initially use the Plus icon to draw the eye. In the Paste page, I see two icons (Plus and ArrowForward)as an example. which icon should I use? 
<img width="676" alt="icons" src="https://user-images.githubusercontent.com/67388401/116733423-f0845d00-a9a0-11eb-9fc8-a4698732fd0b.png">

**Reply by serifluous** [reply]
You'll only need to add the ArrowForward in Figma if you end up using Button as a link/anchor: https://paste.twilio.design/components/button/#buttons-with-link-functionality

Otherwise, the plus button works just great 👍
